mscore/revision.h

```diff
@@ -1 +1 @@
-3543170
+57d3883
```

Comment:
OK, I will change that - it isn't in https://musescore.org/en/developers-handbook/compilation/compile-instructions-linux-bsd-git ...